satellite/console/consoleweb/server.go

```diff
@@ -47,7 +47,7 @@ var (
 
 // Config contains configuration for console web server
 type Config struct {
-	Address         string `help:"server address of the graphql api gateway and frontend app" default:"127.0.0.1:8081"`
+	Address         string `help:"server address of the graphql api gateway and frontend app" devDefault:"127.0.0.1:8081" releaseDefault:"0.0.0.0:10100"`
```

Comment:
Do we really want to expose these things by default rather than let devops manually specify them? It seems like something that we shouldn't do, but I'm not too strongly against it either.

Reply:
I second that, this should stay on localhost

Reply:
the devops config still specifies this config value directly, but i thought if devops wanted it this way maybe other satellite operators would benefit from the default. our default server address is already :7777

Reply:
maybe this shouldn't specify 0.0.0.0 though if that messes up ipv6

Reply:
The default server is password protected so there's less concern for it.

Reply:
the default grpc server? no it's not